src/schemaToCSVTransforms.js

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash/fp'
+import { updateMany, ensureKeys } from './futil'
+import F from 'futil'
+
+// Maps contexture schemas to tranforms for fast-csv
+export let schemaToCSVTransforms = (schema, {logger = _.noop, header = true, include, displayDefault=_.identity} = {}) => {
+  let count = 0
+  let transformedHeaders =  include && F.arrayToObject(_.identity,v=>_.getOr(_.startCase(v),`${v}.label`,schema),include)
```

Comment:
SUPER NITPICKY: It's weird that duti hasn't formatted these files properly, but can you just go through and format the files?  This and the top level function signature should be multiple lines.